openml/datasets/dataset.py

```diff
@@ -297,6 +297,10 @@ def __eq__(self, other):
             "url",
             "dataset",
```

Comment:
Could you please remove this line? There's no attribute `dataset` on the `OpenMLDataset`.

Reply:
Yep, removed what was previously there. I guess someone mistook it for `_dataset`.